tests/test_session.py

```diff
@@ -266,8 +264,8 @@ async def test_session_communication_with_a_tcp_echo_server(src: str, dest: str,
 
     assert "".join(expected) == actual
 
-    assert await src_peer.api.session_close_client(protocol="tcp", bound_ip="127.0.0.1", bound_port=src_sock_port)
-    assert await src_peer.api.session_list_clients("tcp") == []
+    assert await src_peer.api.session_close_client(session) is True
+    assert len(await src_peer.api.session_list_clients(Protocol.TCP)) == 0
```

Comment:
This is probably incorrect merge conflict resolve
- the `is True` is not necessary
- `assert  x == []` is better than `assert len(x) == 0`

Reply:
`is True` is necessary, as if the result of the api call is a float|int != 0, or a non empty list, `assert result` would still be ok, even if the return value isn't `True`.